.github/workflows/verify-templating.yml

```diff
@@ -0,0 +1,22 @@
+name: Verify Templating
+
+on:
+  pull_request:
+  push:
```

Comment:
Since this is merely just creating the folder structure, it would be good to have this on every push so a developer can easily identify in which push did the change stop working. That would be very easily identifiable by just looking at the check mark besides the commit.

Reply:
Main difference is that `push:` will happen on _every_ push where as `pull_request:` will happen on every push that's part of a pull request. keeping it to just commits that are part of a PR can help mitigate cost of CI minutes. So if one's working on something, setting required anyways and that point CI would start verifying templating

Reply:
Removed the `push` and only kept the `pull_requests`.